app/pages/sections/catalogue-solutions/order-item/contextCreator.js

```diff
@@ -1,11 +1,46 @@
 import manifest from './manifest.json';
 import { baseUrl } from '../../../../config';
 
-export const getContext = ({
-  orderId, solutionName, serviceRecipientName, odsCode,
-}) => ({
-  ...manifest,
-  title: `${solutionName} ${manifest.title} ${serviceRecipientName} (${odsCode})`,
-  deleteButtonHref: '#',
-  backLinkHref: `${baseUrl}/organisation/${orderId}/catalogue-solutions/select/solution/recipient`,
+export const addData = ((selectedPrice) => {
```

Comment:
this function isn't returning anything. Is it working as expected?

Reply:
yeah it's just adding to the manifest
all the data gets populated still

Reply:
ah ok. Can we break this down into 2 separate functions. One to check the estimation period and the other one to addData to the table.